data/172/939/597/1/1729395971.geojson

```diff
@@ -27,24 +27,29 @@
     ],
     "src:geom":"uscensus",
     "src:lbl_centroid":"mapshaper",
+    "src:population":"uscensus",
+    "src:population_date":"2023",
     "uscensus:countyfp":135,
+    "uscensus:funcstat":"A",
     "uscensus:geoid":1713526904,
     "uscensus:name":"Fillmore Consolidated",
     "uscensus:namelsad":"Fillmore Consolidated township",
+    "uscensus:population":810,
+    "uscensus:population_date":"2023",
     "uscensus:statefp":17,
     "wof:belongsto":[
         102191575,
         85633793,
-        85688697,
-        102080819
+        102080819,
+        85688697
     ],
     "wof:breaches":[],
     "wof:concordances":{
-        "uscensus:geoid":"1713526904"
+        "uscensus:geoid":"1713526094"
```

Comment:
Changes look good, but just curious, how did you find these mismatches?

Reply:
**tl;dr** – it's really useful to view and interact with the WOF data visually in QGIS, not just as a table in a database or as a text file!

I'm writing a blog post about how to join thematic data tables with WOF using the new official concordance key/value in the SHP distributions. I first noticed that CT counties and localadmin weren't joining, and then immediately north of there in Amherst MA I noticed a localadmin wasn't matching (while 99% of others were matching fine). The CT problem was because the geoid had changed because of US Census lifecycle rules that we needed to catchup to. The MA problem was because we somehow applied the wrong geoid to the localadmin (like sometimes we applied the locality geoid to the parent localadmin). I then downloaded the US Census COUSUB shapefiles for several dozen states from Tiger 2023, merged them, did a spatial join between that and the WOF SHP distro, and evaluated if the features were a good match at all (99.9% but the usual funk around math centroids versus revgeo centroids), and for 17 records that appeared to be exactly the same, I found the wrong geoids. Upon fixing them up, the population geoid table also then joined successfully to achieve a backfill.